### ISSUE TITLE
Make Issues Easier

### ISSUE THREAD

**Issue by mvogelgesang** [GSA/EDX#154]
## Summary

Look into Github Actions as a means for facilitating labeling and organizing issues.

## Additional context and links

Folks on the team have a varying depth of experience with Github which can make the creation and management of issues to feel overly burdensome. 

## Checklist

List below the specific actions to be taken

- [x] Can GH Actions be used to organize labels and projects?
- [x] If super-simple, implement it. If more complex, create new issue

**Comment by mvogelgesang** [comment]
GHA can be triggered by various activities at various lifecycle stages (create (C), edit (E), delete (D)) including Pull Requests, Forks, Issue (CED), Comment (CED), and Label (CED).

An existing action looks to be available that may meet this need:
- https://docs.github.com/en/actions/managing-issues-and-pull-requests/moving-assigned-issues-on-project-boards
- https://github.com/marketplace/actions/github-project-automation

Suggest doing a bit more research with the team to determine what initial automations we would like and create a new task to get those put into place. 

- [ ] Discuss with team automation/ ways to put some bumpers on the issue creation process, document and create new issue.

**Comment by mvogelgesang** [comment]
@anatecture @AKM399 @ryanwoldatwork I merged #165 which established a lightweight action which takes any new issue, assigns it to the Inbox status and to the EDX Backlog project. The total time to complete is about 15 seconds. Next time we all meet as a team, I'll check in to see if there are other steps we can automate- once that is done, I'll plan to close this ticket.

@RachelF @monicas-d @gailsprinkle @hatfieldjm4 >> FYA - Newly created issues are now automatically placed into the project and correct status (give it about 15 seconds so see it appear in the correct spot)

**Comment by mvogelgesang** [comment]
Met with team and described current GitHub Action - no further action needed at this time.
